package.json

```diff
@@ -194,5 +195,10 @@
     "npm": ">=6",
     "pnpm": ">=8"
   },
-  "packageManager": "pnpm@9.15.4+sha256.9bee59c7313a216722c079c1e22160dea7f88df4e0c3450b1d7b01b882336c6a"
+  "packageManager": "pnpm@9.15.4+sha256.9bee59c7313a216722c079c1e22160dea7f88df4e0c3450b1d7b01b882336c6a",
+  "devDependencies": {
+    "@cypress/code-coverage": "^3.14.5",
+    "babel-plugin-istanbul": "^7.0.0",
+    "nyc": "^17.1.0"
+  }
```

Comment:
These should be installed as dependencies instead of dev dependencies for consistency with how all other packages are installed in this workspace root.